docs/sql-programming-guide.md

```diff
@@ -1492,6 +1492,19 @@ that these options will be deprecated in future release as more optimizations ar
   </tr>
 </table>
 
+## Broadcast Hint for SQL Queries
```

Comment:
let's also mention the behavior change in migration guide section.

Reply:
+1